apps/expo/src/components/button.tsx

```diff
@@ -1,27 +1,42 @@
 import * as React from 'react';
+import type { ViewStyle } from 'react-native';
 import { Text, TouchableOpacity } from 'react-native';
 
 type Props = {
     title: string;
     onPress: () => void;
+    colorIndex: number;
+    buttonStyles?: ViewStyle;
 };
 
-export const Button = ({ title, onPress }: Props) => {
+const getPrimaryColor = (colorIndex: number, multiplier: number) => {
```

Comment:
note: you can simplify this to 
`const getPrimaryColor = (colorIndex: number, multiplier: number) => 
  Math.min(colorIndex * multiplier, MAX_COLOR_INTENSITY);`